src/sentry/static/sentry/app/data/forms/projectGeneralSettings.jsx

```diff
@@ -195,6 +195,10 @@ export const fields = {
   scrapeJavaScript: {
     name: 'scrapeJavaScript',
     type: 'boolean',
+    disabled: ({organization, name}) => !organization[name],
```

Comment:
Maybe add a comment saying if this turned OFF in org settings, then you cannot override in project settings.